eth_tester/utils/backend_testing.py

```diff
@@ -942,7 +942,9 @@ def test_log_filter_picks_up_new_logs(self, eth_tester, filter_topics, expected)
 
         specific_logs_changes = eth_tester.get_only_filter_changes(filter_event)
         specific_logs_all = eth_tester.get_all_filter_logs(filter_event)
-        assert len(specific_logs_changes) == len(specific_logs_all) == expected
+        specific_direct_logs_all = eth_tester.get_logs_directly(topics=filter_topics)
+        assert len(specific_logs_changes) == len(specific_logs_all) == \
```

Comment:
Now that this is overflowing the line, can we change this to be three separate assertions for easier readability.

Reply:
No problem.